apps/condo/domains/ticket/schema/UserTicketCommentRead.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Generated by `createschema ticket.UserTicketCommentRead 'user:Relationship:User:CASCADE; ticket:Relationship:Ticket:CASCADE; readResidentCommentAt:DateTimeUtc;'`
+ */
+
+const { Text, Relationship, Integer, Select, Checkbox, DateTimeUtc, CalendarDay, Decimal, Password, File } = require('@keystonejs/fields')
+const { Json } = require('@core/keystone/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/ticket/access/UserTicketCommentRead')
+
+
+const UserTicketCommentRead = new GQLListSchema('UserTicketCommentRead', {
+    schemaDoc: 'Last date the comment was read in the ticket by the user',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        user: {
+            schemaDoc: 'The user who read the comment',
+            type: Relationship,
+            ref: 'User',
+            isRequired: true,
+            knexOptions: { isNotNullable: true },
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+        ticket: {
+            schemaDoc: 'Ticket in which the user read the comment',
+            type: Relationship,
+            ref: 'Ticket',
+            isRequired: true,
+            knexOptions: { isNotNullable: true },
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+        readResidentCommentAt: {
```

Comment:
This name confuses with name of the entity `UserTicketCommentRead`. From this set of fields it seems, like only resident reading actions will be tracked, but entity named in more generic form `User…`